### PR TITLE
Release 0.48.0

### PR DESCRIPTION
**Pull request by qinqon**
**What this PR does / why we need it**:
Release 0.48.0

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by qinqon** [comment]
/override pull-e2e-cluster-network-addons-operator-nmstate-functests

known flaky

File: version/description [hunk 1]
```diff
@@ -1,10 +1,5 @@
...
-* Bump CNI plugins version (#772)
-* bump nmstate to v0.40.0 (#769)
-
-Bugs:
-* Don't run etcd in memory by default (#776)
+0.48.0
 
+* bump nmstate to v0.42.0 (#792)
+* bump kubemacpool to v0.23.0 (#788)
+* bump ovs-cni to v0.19.1 (#786)
```
Comment (RamLavi) [inline version/description:5]: can you manually add #795 (and #796 if it's merged) ?

**Review by RamLavi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**